pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -18,6 +18,8 @@ type KeycloakRealmSpec struct {
 	// +kubebuilder:validation:Required
 	Realm *KeycloakAPIRealm `json:"realm"`
 	// A list of overrides to the default Realm behavior.
+	// +listType=map
+	// +listMapKey=identityProvider
```

Comment:
This one has been addressed in https://github.com/keycloak/keycloak-operator/pull/282, so we probably need to wait till it gets merged and this PR gets rebased.

Reply:
Sure, the commit is only only included because that produced a warning during code generation.